docs/edgeql/sets.rst

```diff
@@ -248,6 +248,43 @@ Use the :eql:op:`union` operator to merge two sets.
   db> select {1, 2} union {3.1, 4.4};
   {1.0, 2.0, 3.1, 4.4}
 
+Finding common members
+----------------------
+
+.. versionadded:: 3.0
+
+Use the :eql:op:`intersect` operator to find common members between two sets.
+
+.. code-block:: edgeql-repl
+
+    db> select {1, 2, 3, 4, 5} intersect {3, 4, 5, 6, 7};
+    {3, 5, 4}
+    db> select {'a', 'b', 'c', 'd', 'e'} intersect {'c', 'd', 'e', 'f', 'g'};
+    {'e', 'd', 'c'}
+
+.. note::
+
+    The ordering of the returned set may not match that of the operands.
+
+
+Finding unique members
```

Comment:
Surely, it's not about finding unique members as much as removing certain subset from the original set.